tests/admin_inlines/models.py

```diff
@@ -56,6 +56,10 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class NonAutoPKBookDescendant(NonAutoPKBook):
```

Comment:
I'd use Child rather than Descendant as this seems like more common terminology in the test models.